Lib/test/test_pathlib/test_pathlib.py

```diff
@@ -129,6 +129,9 @@ class StrSubclass(str):
         for part in p.parts:
             self.assertIs(type(part), str)
 
+    def test_lazy_import(self):
```

Comment:
Would you mind adding the `@cpython_only` decorator, since these tests don't necessarily make sense on other Python implementations?

I should have done that on my PRs too but didn't think about it.

Reply:
Done.